bin/ansible-connection

```diff
@@ -293,6 +293,15 @@ def main():
 
         else:
             messages.append('found existing local domain socket, using it!')
+            conn = Connection(socket_path)
+            pc_data = to_text(init_data)
+            try:
+                messages.extend(conn.update_play_context(pc_data))
+            except ConnectionError as exc:
```

Comment:
i thought `utils/jsonrpc.py` will detect missing method and respond accordingly.  If not, we should fix it there, imho

Reply:
I think ConnectionError (with a specific, identifiable err code) _is_ the right response here, rather than some other, potentially surprising default.